src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,20 @@
+package duke.parser;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import duke.exception.DukeException;
+import duke.tasklist.TaskList;
+
+public class ParserTest {
+    @Test
+    public void byeTest() {
+        Boolean result = false;
```

Comment:
Small nit, `Boolean` refers to the Object type boolean while `boolean` refers to the primitive type. Consider using the primitive type instead. 👍